samples/nrf9160/lte_ble_gateway/README.rst

```diff
@@ -90,8 +90,8 @@ After programming the main controller with the sample, you can test it as follow
 #. Open a web browser and navigate to https://nrfcloud.com/.
    Follow the instructions to set up your account and to add an LTE device.
    A pattern of switch and button actions is displayed on the webpage.
-#. Power on or reset the board.
-#. Observe in the terminal window connected to the first serial port that the board starts up in the Secure Partition Manager.
+#. Power on or Reset the kit.
```

Comment:
This line looks confusing to me.
Does the capitalization allude to PCB button/switch?
Does reset mean factory reset?
Maybe this would be enough: ``Power on the kit (or restart it).``?

Reply:
That was an automatic replace - should be lowercase.
Will fix. :)

And in general, the reason for telling people to reset is that you'll usually have the kit turned on already, so you'll just skip this line - but you must reset/restart it or it won't work. And you do this with the RESET button.